### PR TITLE
SALTO-4658: add important values to zendesk types

### PR DESCRIPTION
**Pull request by edenhassid**
add important values to zendesk types

---

_Additional context for reviewer_

---
_Release Notes_: 
zendesk:
* add important values annotation to types

---
_User Notifications_: 
None

### CODE REVIEW

File: packages/zendesk-adapter/src/filters/add_important_values.ts [hunk 1]
```diff
@@ -0,0 +1,571 @@
...
+      value: 'sorting',
+      highlighted: false,
+      indexed: true,
+    },
+  ],
+}
+
+const filterCreator: FilterCreator = () => ({
+  name: 'addImportantValues',
+  onFetch: async (elements: Element[]): Promise<void> => {
```
Comment (netama) [inline packages/zendesk-adapter/src/filters/add_important_values.ts:559]: what's the plan around generalizing? can we later move this filter to adapter-components?

File: packages/zendesk-adapter/src/filters/add_important_values.ts [hunk 2]
```diff
@@ -0,0 +1,571 @@
...
+    const objectTypes = elements.filter(isObjectType)
+    objectTypes.forEach(obj => {
+      const { typeName } = obj.elemID
+      const importantValuesArray = importantValuesMap[typeName]
+      if (_.isArray(importantValuesArray)) {
```
Comment (netama) [inline packages/zendesk-adapter/src/filters/add_important_values.ts:564]: nit: we prefer builtins when available (`Array.isArray`)

**Review by netama — APPROVED** [review]
didn't compare the cases to the table, but looks good code-wise :)

File: packages/zendesk-adapter/src/filters/add_important_values.ts (hunk 1 above)
Comment (edenhassid) [reply packages/zendesk-adapter/src/filters/add_important_values.ts:559]: discussed IRL, we will consider in the future if we want to move it to adapter-components

**Pull request merged**